DXRMapFixups/DeusEx/Classes/DXRFixupM06.uc

```diff
@@ -104,6 +104,18 @@ function PreFirstEntryMapFixes()
                     break;
                 }
             }
+
+            foreach AllActors(class'Button1', b) {
+                if (b.tag == 'Weapons_Lock_broken' || b.tag == 'Missile_door_toggle' || b.tag == 'Weapons_lock') {
+                    b.SetRotation(rot(14312, 18536, 51312)); // my guess is that the vanilla rotations were each done indiviually by eye
```

Comment:
Only a comment, really.  In the vanilla unmirrored map, this essentially matches the "Munitions Bay" button on the non-Jock side, except the roll is inverted:

![image](https://github.com/Die4Ever/deus-ex-randomizer/assets/13684088/aed49f58-38ea-4dd7-8555-ec8367cce3ce)

This shouldn't really matter, since the buttons don't move.  It would arguably be vaguely more clean to have it match a non-mirrored button, but I don't care much

Reply:
I know they probably have the same rotation either way, but still should probably be using rotm for these instead of rot? I forget the exact format for the 4th parameter but you can find examples elsewhere

```suggestion
                    b.SetRotation(rotm(14312, 18536, 51312)); // my guess is that the vanilla rotations were each done indiviually by eye
```

Reply:
Using `rotm` instead of `rot` does not have the desired effect.
![image](https://github.com/Die4Ever/deus-ex-randomizer/assets/17994307/05a59bb3-0827-4f3e-893a-fca2e097617f)

Reply:
That's also true for the other "Munitions Bay" button, when seen using legend. When logged though, they show the value I used. 65535 (360 degrees) minus 51312 is 14223, so it makes sense.